TimeIsMoney/TimeIsMoneyMasterViewController.h

```diff
@@ -7,23 +7,31 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "AppDelegate.h"
 
-#import "TimeIsMoneySettingsModel.h"
+typedef enum timerState
+{
+    STOPPED,
+    PAUSED,
+    RUNNING_TASK,
+    RUNNING_BREAK,
+} TimerState;
 
 @interface TimeIsMoneyMasterViewController : UIViewController
 {
+    TimerState previousState;
+    TimerState timerState;
     @private
     NSTimer *countdownTimer;
     int remainingTicks;
     int pauseTime;
-    BOOL isCountdownTimerPaused;
 }
 
 @property (weak, nonatomic) IBOutlet UILabel *timeLabel;
 @property (weak, nonatomic) IBOutlet UIButton *playButton;
 @property (weak, nonatomic) IBOutlet UIButton *stopButton;
 @property (weak, nonatomic) IBOutlet UIButton *resetButton;
-@property (weak, nonatomic) TimeIsMoneySettingsModel *settings;
+@property (weak, nonatomic) AppDelegate *delegate;
```

Comment:
You don't have to hang on to a variable to the appdelegate. just get it as needed.